core-tests/shared/src/test/scala/zio/FiberSpec.scala

```diff
@@ -162,7 +162,38 @@ object FiberSpec extends ZIOBaseSpec {
             currentFiber <- ZIO.succeed(Fiber.currentFiber())
           } yield assertTrue(currentFiber.isDefined)
         }
-      } @@ TestAspect.fromLayer(Runtime.enableCurrentFiber)
+      } @@ TestAspect.fromLayer(Runtime.enableCurrentFiber),
+      suite("FiberFailure stack trace handling")(
```

Comment:
I think this can be moved in its own test file (`FiberFailureSpec`) as it's not really a test for `Fiber`